src/pudl/convert/censusdp1tract_to_sqlite.py

```diff
@@ -152,7 +175,9 @@ def main():
 
     pudl_settings["sandbox"] = args.sandbox
 
-    censusdp1tract_to_sqlite(pudl_settings=pudl_settings, ds=ds, clobber=args.clobber)
+    _ = censusdp1tract_to_sqlite(
```

Comment:
Did you want to use this assignment somewhere else?

Reply:
This just returns the path to the saved DB, which we don't need to call anywhere else. The loggers for this function already print the output path, which would be the only possible application I could think of for this variable.

Reply:
Hmm, at this point could you just avoid the assignment altogether?

Reply:
Indeed, I deleted it (and the whole CLI interface)! :)